app/src/main/java/org/p2p/wallet/home/ui/main/empty/GetTokenViewHolder.kt

```diff
@@ -0,0 +1,26 @@
+package org.p2p.wallet.home.ui.main.empty
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.p2p.wallet.databinding.ItemGetTokenBinding
+import org.p2p.wallet.home.model.HomeElementItem
+import org.p2p.wallet.home.ui.main.adapter.OnHomeItemsClickListener
+
+class GetTokenViewHolder(
```

Comment:
I would name it as `PopularTokenToGetViewHolder` / `PopularTokenViewHolder`
because it's the list of popular tokens really